changelog.md

```diff
@@ -13,6 +13,12 @@ Since you are interested in what happens next, in case, you work for a for-profi
 
 ---
 
+## 10.0.0-alpha.24 (2019-8-13)
```

Comment:
we have a Next headline which will be replaced with the release version at release time

Reply:
so below Next is the next release changes